Gulpfile.js

```diff
@@ -1,5 +1,4 @@
 /* eslint-disable */
-'use strict';
```

Comment:
just a note: we don't need `use strict` here.  this was removed at my request (thanks for doing that, btw)